app/templates/src/app.js

```diff
@@ -1,18 +1,16 @@
-import Map from "can/map/";
-import route from "can/route/";
-import 'can/map/define/';
-import 'can/route/pushstate/';
+import DefineMap from 'can-define/map/';
+import route from 'can-route';
+import 'can-route-pushstate';
 
-const AppViewModel = Map.extend({
-  define: {
-    message: {
-      value: 'Hello World!',
-      serialize: false
-    },
-    title: {
-      value: '<%= name %>',
-      serialize: false
-    }
+const AppViewModel = DefineMap.extend({
+  route: "string",
```

Comment:
@leoj3n this is a part of the DefineMap (can-define plugin), which allows you to define attributes and their types ("string" in this case) and gives you observable attributes without having to use `.attr()`.  So you can just use the normal js dot operator to change properties of a DefineMap instance.